extensions/wikia/AdEngine3/AdEngine3.class.php

```diff
@@ -115,8 +115,6 @@ public static function getContext( Title $title = null ) {
 			return [
 				'bidders' => [],
 				'opts' => array_filter([
-					'adEngineVersion' => $wg->AdEngineVersion,
```

Comment:
Correct me if I'm wrong but I think we use it for tracking ad-engine version.

Reply:
Disagree. There are two usages of this variable in mobile-wiki. First for reporting, but this value comes from `window.ads` and it is populated by ad-engine itself:
https://github.com/Wikia/mobile-wiki/blob/78505e69dfd462a39aec4f4f932b68a473bf9468/app/modules/ads/core/stable-ads.js#L109
Second is used in adengine-experimental feature and this part of code will be unused:
https://github.com/Wikia/mobile-wiki/blob/e82bd3b86e4d6ee03e4bf937e50ef2f75fe1b02a/app/modules/ads/core/experimental-ads.js#L33
Please review.